lib/rqrcode/qrcode/qr_code.rb

```diff
@@ -18,6 +18,13 @@ module RQRCode #:nodoc:
     :mode_kanji         => 1 << 3
   }
 
+  QRMODE_NAME = {
+    :number        => :mode_number,
+    :alphanumeric  => :mode_alpha_numk,
+    :'8bit_byte'   => :mode_8bit_byte,
+    :kanji         => :mode_kanji
```

Comment:
What do say about leaving kanji out from the list of modes. Its not supported so i am thinking i might just lead to misunderstandings.